src/main/resources/templates/editEvent.html

```diff
@@ -19,11 +19,7 @@
         <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
         <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
         <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
-        <script type="text/javascript" th:inline="javascript">
-            $(function () {
-                $("#searchTerm").hide()
-            });
-        </script>
+        <script type="text/javascript" th:inline="javascript" src="hide.js"></script>
```

Comment:
Normally I would leave this as is, but it is used in two html files.
I moved it to it's own javascript file. This is just in case you want to expand upon it later, and dont want to copy and paste
https://www.baeldung.com/java-clean-code

Reply:
It's a good idea to keep JavaScript in a separate file.  Keep content, styling (css), and behavior (js) separate.